packages/ibm-products/package.json

```diff
@@ -93,9 +93,12 @@
     "@carbon/themes": "^10.55.5",
     "@carbon/type": "^10.45.5",
     "carbon-components": "^10.59.0",
-    "carbon-components-react": "^7.70.0",
+    "carbon-components-react": "^7.60.0",
     "carbon-icons": "^7.0.7",
     "react": "^16.8.6 || ^17.0.1",
     "react-dom": "^16.8.6 || ^17.0.1"
+  },
+  "resolutions": {
```

Comment:
Just wondering about using resolutions here? If we need it, we'll need to move it to the root package.json

Reply:
💀

I was typing out to demo. Sorry, I’m a mess this morning 😭